bootstrap/config.yaml

```diff
@@ -2,30 +2,31 @@ Server:
   ims_host: ims-na1.adobelogin.com
   ims_endpoint_jwt: /ims/exchange/jwt
 
+# All the Enterprise values can be obtained from the Adobe IO Integration
 Enterprise:
-  api_key: af80375bec544eb493ed271373b12191
-  org_id: 20656D0F5B9975B20A495E23@AdobeOrg
-  tech_acct: F98D7E725CCA6F770A495FD9@techacct.adobe.com
+  api_key: <api_key>
+  org_id: <org_id> # Sample value for reference - 20656D0F5B9975B20A495E23@AdobeOrg
+  tech_acct: <technical_account_id> # Sample value for reference - F98D7E725CCA6F770A495FD9@techacct.adobe.com
   client_secret: <client_secret>
-  priv_key_filename: private.key
+  priv_key_filename: <private_key_filename>
 
 Platform:
   platform_gateway: https://platform.adobe.io
-  ims_token: None
+  ims_token: <ims_token> # ims_token should be prefixed with "Bearer "
```

Comment:
feels ambiguous. Should the user insert "Bearer " ? If so, perhaps they shouldn't have to, and if they still do we would still do the right thing 😊

Reply:
actually, after reviewing the code it seems you are adding Bearer if it's not passed.